src/GitHub.VisualStudio/Views/GitHubPane/PullRequestDetailView.xaml

```diff
@@ -2,47 +2,52 @@
                                     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                                     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-                                    xmlns:ghfvs="https://github.com/github/VisualStudio"
```

Comment:
Should we be replacing this? Isn't it the new style of XAML reference that was recently merged?

Reply:
Oops, merge/rebase error!